### DISCUSSION TITLE
thumbnail script isn't loading on v.37

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#13937] [Q&A] [answered]
Hi I use a script for youtube thumbnails. On v36 the script works fine but on v.37 it isn't loading.
Thanks for any help :)

logfile
[output.txt](https://github.com/mpv-player/mpv/files/15041764/output.txt)

cmd sceenshot
![jhgjhgjh](https://github.com/mpv-player/mpv/assets/139760266/843c67fc-970a-43a6-b56d-a4810499311f)

this is the profile 
```
[YouTube]
profile-cond=demuxer_via_network and mp.command('load-script "C:/Program Files (x86)/MPV/gucio/mpv_thumbnail_script_server.lua"; load-script "C:/Program Files (x86)/MPV/gucio/mpv_thumbnail_script_client_osc.lua"')
profile-restore=copy
```

**Comment by llyyr** [comment]
https://github.com/mpv-player/mpv/commit/86b498ecc0e6276bb905df57f479d0506b45ba53

**Reply by geextahslex** [reply]
okay this script https://github.com/TheAMM/mpv_thumbnail_script was updated the last time 5 years ago, so yeah... 
Does anyone know a replacement?

**Reply by geextahslex** [reply, marked as answer]
https://github.com/marzzzello/mpv_thumbnail_script

**Reply by kasper93** [reply]
Isn't https://github.com/po5/thumbfast also good?

**Reply by geextahslex** [reply]
It is pretty bad at youtube thumbnails and this is the only thing I need it for

**Reply by po5** [reply]
One day I'll resume work on it... :pensive:

**Reply by yuka123** [reply]
Is there a fix for Thumbfast for > 0.36.x ? Anybody has updated lua
